server/routes/api/accommodations/accommRoute.js

```diff
@@ -0,0 +1,45 @@
+import express from 'express';
+import accomController from '../../../controllers/accomodationController';
+import accommodationValidation from '../../../middlewares/validators/accommodation';
+import { authorizationCheck } from '../../../middlewares/authorization';
+import allow from '../../../middlewares/roleAuthorisation';
+
+const router = express.Router();
+
+router.post(
+  '/',
+  authorizationCheck,
+  allow('Travel Administrator', 'Supplier'),
+  accommodationValidation.validateAccommodationData,
+  accomController.createAccommodation,
+);
+
+router.get(
+  '/:id',
+  authorizationCheck,
+  allow('Travel Administrator', 'Supplier'),
+  accommodationValidation.validateIdFromParams,
+  accommodationValidation.checkAccommodationExist,
+  accomController.getSpecificAccomDetails,
+);
+
+router.get(
+  '/city/accommodations/:city',
```

Comment:
Here I think the overall route would be `/api/v1/accommodations/city/accomodations/:city` what if we could use `/api/v1/accommodations/city/:city`